vtex-apps/store-graphql
# Handle Assembly Options' Input Values

#### How should this be manually tested?

1. Open [Workspace](https://breno--storecomponents.myvtex.com/star-color-top/p)
2. Choose a Font
3. Check Glossy Print
4. Click Add to cart
5. Go to the checkout page

Type in the console: `vtexjs.checkout.orderForm.items[0].assemblies[0]`

You should see the assemblies are added:
![image](https://user-images.githubusercontent.com/284515/65174918-38eb4300-da28-11e9-9e91-00a21e1027b7.png)


#### Type of changes


✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
_ | Technical improvements 

#### Notes
